functions/v2/pubsub/subscribe-to-topic/src/main/java/functions/SubscribeToTopic.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions;
```

Comment:
This usually follow the groupId pattern: `com.example.cloud.functions.v2` but I see that the other samples don't follow that idiom.

Reply:
`package v2` feels a bit weird to me.

I'm fine with `package functions.v2`, but otherwise I'd vote for leaving it as-is.

Reply:
I'll implement it as `package functions.v2`.